code-challenges.test.js

```diff
@@ -49,8 +86,33 @@ const studyMinutesWeek2 = {
 }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
+// describe('arrSort', () => {
+//     it("gives an array of numbers from least to greatest", () => {
+//       expect(arrSort(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+//       expect(arrSort(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+//     })
+//   })
+  // ReferenceError: arrSort is not defined
+
+// pseudocode ->
+// input - an object
+// output - array with the object values sorted from least to greatest.
+
+// process ->
+// create a function named arrSort that will receive an object as an input and extract its values
+// using the Object.values method to get all values from the object and turn into an array
+// the sort' method will sort the final array using the comparison function that subtracts a from b .
```

Comment:
It does subtract a from b, but it uses the result of that subtraction to determine where each value goes in reference to the other value!